toolkit/tools/pkg/imagecustomizerlib/versionsOfToolDependencies.go

```diff
@@ -0,0 +1,95 @@
+package imagecustomizerlib
+
+import (
+	"os/exec"
+	"regexp"
+	"strings"
+
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/logger"
+)
+
+func getVersionsOfToolDeps() {
```

Comment:
This function isn't returning anything. So, it should probably be named something like `logVersionsOfToolDeps`.

Reply:
Updated the function name.